### PR TITLE
Fix Solarized Light theme

### PR DESCRIPTION
**Pull request by zeitgeist87**
The official color palette is included as a CSS comment and every
color used is named with a comment.

### CODE REVIEW

**Comment by zeitgeist87** [comment]
It definitely needs a bit of tweaking. I am not very good at themes, so I'm open for suggestions on how to improve it.

**Comment by zeitgeist87** [comment]
The result looks like this:

solarized

**Comment by LeaVerou** [comment]
Thanks!

The main problem I see is that the default color is too light. We should use the darkest color there.

**Comment by LeaVerou** [comment]
For anyone interested in helping, here is a comparison. First image is Solarized Light in my editor, second is Prism from the website.

image
image

Ours looks a bit more colorless, but overall, not too bad actually!!
Thanks again @zeitgeist87!!

**Comment by mAAdhaTTah** [comment]
That may be a function of the syntax highlighting rather than the theme. You'll notice in the editor `tokens.content` is two different colors, whereas they're both the same in Prism. Also, in the editor, `tokens.map` (e.g. {object}.{method}) is highlighted differently than  `tokens.replace`. Not sure more can be done without changing Prism itself. 

Also, wondering if a new version of Prism could be pubbed to npm. Would love to get this theme into my plugin ☺.

**Comment by zeitgeist87** [comment]
> Not sure more can be done without changing Prism itself.

We can quite easily make the punctuation red. But I agree, that some of it is not possible with Prism.

> Also, wondering if a new version of Prism could be pubbed to npm. Would love to get this theme into my plugin ☺.

@mAAdhaTTah I will try to create a new release this weekend. There is a bit of work involved, because the CHANGELOG has to updated.

File: themes/prism-solarizedlight.css [hunk 1]
```diff
@@ -7,29 +7,60 @@
...
 }
 
+pre[class*="language-"]::-moz-selection, pre[class*="language-"] ::-moz-selection,
+code[class*="language-"]::-moz-selection, code[class*="language-"] ::-moz-selection {
+	background: #073642; /* base02 */
+}
+
+pre[class*="language-"]::selection, pre[class*="language-"] ::selection,
+code[class*="language-"]::selection, code[class*="language-"] ::selection {
+	background: #073642; /* base02 */
```
Comment (zeitgeist87) [inline themes/prism-solarizedlight.css]: I would appreciate it, if someone could check if the selection looks right. I am not sure if `base02` is the right color for that.

**Pull request merged**